src/esp/sensor/CameraSensor.h

```diff
@@ -13,9 +13,7 @@ namespace esp {
 namespace sensor {
 
 struct CameraSensorSpec : public VisualSensorSpec {
-  int channels = 4;
-  // description of Sensor observation space as gym.spaces.Dict()
-  std::string observationSpace = "";
+  float ortho_scale = 0.1f;
```

Comment:
I realize you're just moving this ortho_scale field from elsewhere, but how about fixing to be camelCase per our coding standard?